src/Compilers/CSharp/Portable/BoundTree/UnboundLambda.cs

```diff
@@ -390,14 +390,15 @@ internal partial class UnboundLambda
             SeparatedSyntaxList<ParameterSyntax>? syntaxList,
             ImmutableArray<EqualsValueClauseSyntax?> defaultValues,
             bool isAsync,
-            bool isStatic)
+            bool isStatic,
+            bool hasParamsArray)
         {
             Debug.Assert(binder != null);
             Debug.Assert(syntax.IsAnonymousFunction());
             bool hasErrors = !types.IsDefault && types.Any(static t => t.Type?.Kind == SymbolKind.ErrorType);
 
             var functionType = FunctionTypeSymbol.CreateIfFeatureEnabled(syntax, binder, static (binder, expr) => ((UnboundLambda)expr).Data.InferDelegateType());
-            var data = new PlainUnboundLambdaState(binder, returnRefKind, returnType, parameterAttributes, names, discardsOpt, types, refKinds, declaredScopes, defaultValues, syntaxList, isAsync, isStatic, includeCache: true);
+            var data = new PlainUnboundLambdaState(binder, returnRefKind, returnType, parameterAttributes, names, discardsOpt, types, refKinds, declaredScopes, defaultValues, syntaxList, isAsync, isStatic, hasParamsArray, includeCache: true);
```

Comment:
>isAsync, isStatic, hasParamsArray [](http://example.com/codeflow?start=187&length=33)

Consider using named parameters for these `bool` values.

Reply:
Thanks. 94598c48730461150c2aa222861053afdf6b2017